### DISCUSSION TITLE
FLOSIC Forum

### DISCUSSION THREAD

**Discussion by jacks1ka** [FLOSIC/PublicRelease_2020#12] [General]
Welcome to the FLOSIC Forum, a place where users of the FLOSIC code can post information or questions about FLOSIC calculations.  It is a place to share problems you are having and tricks you have learned.  Please use the Forum actively, so that all may learn from the collective experience of the user group.

**Comment by KamalCMU** [comment]
The Natural Bond Orbital (NBO) input file can be created by the NRLMOL code to be used later by the NBO utility program, which produces an output file containing detailed information about the electron occupancies in the studied system. 

  In the NRLMOL source file directory, the (**nbodrv.f**) and (**bldpmat.f**) files contain the relevant subroutines to NBO.
  The NBO user may need to uncomment the following lines (in **main.ftn**) :
  ------------------------------------------------------- 
   IF (PRTMOS.OR.PRTPMAT.OR.NBO) CALL BLDPMAT
      and
   IF (PRTSMAT.OR.NBO) CALL NBODR
 --------------------------------------------------------
Then compile the code. Then in  **NRLMOL_INPUT.DAT** turn on :
------------------------
**MOLDENV       = 'Y'**

**NBOV              = 'Y'**
------------------------
When the calculation ends you will find "**CLUSTER.NBO**". Then run  the NBO command  
 **gennbo CLUSTER.NBO > output.nbo** 
 (the user should check if NBO program is available on his/her machine) then check the **output.nbo** file which should contain the NBO analysis for your system.
